test/unit/generator/get-document-definition.test.js

```diff
@@ -2,41 +2,41 @@ const styles = require('../../../app/generator/styles')
 const getDocumentDefinition = require('../../../app/generator/get-document-definition')
 const mockStatement = require('../../mocks/mock-statement')
 const { A4 } = require('../../../app/generator/page-sizes')
-const { STATEMENT } = require('../../../app/constants/document-types')
+const { STATEMENT, SFI23QUARTERLYSTATEMENT } = require('../../../app/constants/document-types')
 
 describe('get document definition', () => {
   test('includes A4 paper size', () => {
-    const result = getDocumentDefinition(mockStatement, STATEMENT)
+    const result = getDocumentDefinition(mockStatement, STATEMENT, SFI23QUARTERLYSTATEMENT)
```

Comment:
Wrong signature for getDocumentDefinition

Reply:
I have rewritten the logic for the tests